PROVISIONING-METAL.md

```diff
@@ -0,0 +1,169 @@
+```
+
+Next, set your desired version and variant, and download the image:
+```
+VERSION="1.8.0"
+VARIANT="metal-k8s-1.23"
+SHA="a6233c22"
+IMAGE="bottlerocket-${VARIANT}-x86_64-${VERSION}-${SHA}.img.lz4"
+OUTDIR="${VARIANT}-${VERSION}"
+
+tuftool download "${OUTDIR}" --target-name "${IMAGE}" \
+   --root ./root.json \
+   --metadata-url "https://updates.bottlerocket.aws/2020-07-07/${VARIANT}/x86_64/" \
+   --targets-url "https://updates.bottlerocket.aws/targets/"
+```
+
+### User data
+
+For metal, Bottlerocket expects a TOML-formatted file named `user-data.toml` that contains user data settings.
+Acceptable settings can be found in the [settings docs](https://github.com/bottlerocket-os/bottlerocket#settings).
+
+If you're just getting started and want to provision a host without connecting to a kubernetes cluster, you can use the following example user data which will start kubelet in standalone mode.
+
+```toml
+[settings.kubernetes]
+standalone-mode = true
+```
+
+It's important to note for remote access to your running Bottlerocket hosts, you will need to add user data to enable host containers.
+The Bottlerocket metal images don't enable any host containers by default.
+You can use our [admin](https://github.com/bottlerocket-os/bottlerocket-admin-container) and/or [control](https://github.com/bottlerocket-os/bottlerocket-control-container) containers, but they need to be configured first.
+Full configuration details are covered in the [admin container documentation](https://github.com/bottlerocket-os/bottlerocket-admin-container#authenticating-with-the-admin-container) and the [control container documentation](https://github.com/bottlerocket-os/bottlerocket-control-container#connecting-to-aws-systems-manager-ssm).
+
+### Network interface configuration
+
+Bottlerocket provides the means to configure the physical network interfaces in the system via TOML-formatted file `net.toml`.
+Bottlerocket will read this file at boot time and generate the proper configuration files in the correct format for each interface described; no default configuration is provided.
+For now, we support simple DHCP4 and DHCP6 configuration and plan to support additional configuration in the future.
+
+#### `net.toml` structure
+
+The configuration file must be valid TOML and have the filename `net.toml`.
+The first and required top level key in the file is `version`, currently only `1` is supported.
+The rest of the file is a map of interface name to supported settings.
+Interface names are expected to be correct as per `udevd` naming, no interface naming or matching is supported.
+
+#### Supported interface settings
+
+* `primary` (boolean): Uses the interface as the primary network interface, kubelet will use this interface's IP to register.  If none of the interfaces has `primary` set, we choose the first interface in the file.
```

Comment:
```suggestion
* `primary` (boolean): Use this interface as the primary network interface. `kubelet` will use this interface's IP when joining the cluster.  If none of the interfaces has `primary` set, we choose the first interface in the file.
```

Another stylistic note - @stockholmux might be trying to move away from the "we" language especially when talking about implementation mechanics.

Reply:
Thanks @bcressey! I do have opinions in this - back in my previous organization [I wrote an explainer on the use of 'we'](https://gist.github.com/stockholmux/52b9b224ad806acd0dd389fe3e79daa0). 

In the case of above you're use 'we' to refer to the logic in the software, which is actually a separate issue: software anthropomorphism. It's usually easy to fix - anthropomorphism is unneeded and switch to the explicit name: 

> If none of the interfaces has `primary` set, we choose the first interface in the file.

becomes

> If none of the interfaces has `primary` set, `kubelet` will use the first interface in the file.